roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -0,0 +1,633 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# pylint: disable=line-too-long,invalid-name
+
+"""For details on this module see DOCUMENTATION (below)"""
+
+# router/registry cert grabbing
+import subprocess
+# etcd config file
+import ConfigParser
+# Expiration parsing
+import datetime
+# File path stuff
+import os
+# Config file parsing
+import yaml
+# Certificate loading
+import OpenSSL.crypto
+
+DOCUMENTATION = '''
```

Comment:
https://github.com/ansible/ansible/blob/devel/hacking/module_formatter.py is how Ansible generates their module docs, haven't looked to see if it can be used unmodified or would need to be tweaked for our use.